repos/fdbt-site/tests/utils/index.test.ts

```diff
@@ -249,4 +252,66 @@ describe('index', () => {
             expect(result).toBeTruthy();
         });
     });
+
+    describe('fareTypeIsAllowedToAddACap', () => {
+        const capFareTypes = ['single', 'flatFare', 'return'];
+        const unCapFareTypes = ['period', 'multiOperator', 'schoolService'];
+
+        it.each(capFareTypes)('returns true if the fare type is %s', (fareType: string) => {
+            const result = fareTypeIsAllowedToAddACap(fareType);
+            expect(result).toBeTruthy();
+        });
+
+        it.each(unCapFareTypes)('returns false if the fare type is other than %s', (fareType: string) => {
+            const result = fareTypeIsAllowedToAddACap(fareType);
+            expect(result).toBeFalsy();
+        });
+    });
+
+    describe('getUniqueServices', () => {
+        const services: MyFaresService[] = [
+            {
+                id: '1',
+                lineName: '1',
+                lineId: '4YyoI0',
+                origin: 'Ballarat west',
+                destination: 'Florinas North',
+                startDate: '05/04/2020',
+                endDate: '25/04/2020',
+            },
+            {
+                id: '2',
+                lineName: '2',
+                lineId: 'YpQjUw',
+                origin: 'Ballarat East',
+                destination: 'Florinas',
+                startDate: '05/04/2020',
+                endDate: '25/04/2020',
+            },
+            {
+                id: '3',
+                lineName: '2',
+                lineId: 'YpQjUw',
+                origin: 'Ballarat East',
+                destination: 'Florinas',
+                startDate: '05/04/2020',
+                endDate: '25/04/2020',
+            },
+            {
+                id: '4',
+                lineName: '3',
+                lineId: 'Jk79kC',
+                origin: 'Vega de Pas',
+                destination: 'Guiddam',
+                startDate: '05/04/2020',
+                endDate: '25/04/2020',
+            },
+        ];
+
+        it('returns unique services if two services having same lineName, startDate and endDate ', () => {
+            const result = getUniqueServices(services);
+            expect(services.length).toBeLessThanOrEqual(4);
```

Comment:
We should be checking they return the services we expect, not just checking the length.

Reply:
And we also need a test to make sure it doesnt change anything if there are no duplicates.